docs/reference/cli.md

```diff
@@ -344,14 +350,6 @@ Rebuild a package from a package file instead of a recipe
 	The package file to rebuild
 
 
-- `--no-test`
```

Comment:
unfortunately removing this option will break conda-forge. We should keep it but deprecate it.

Reply:
ok sounds good.